### ISSUE TITLE
Accessibility: "Certain ARIA roles must be contained by particular parents (aria-required-parent)"

### ISSUE THREAD

**Issue by LunaWanderer1** [jquense/react-big-calendar#2394]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/s/react-big-calendar-example-forked-eztu2h

### Bug description

Testing the calendar with jest-axe is rendering the following issue:

```
Expected the HTML found at $('.rbc-off-range.rbc-date-cell[role="cell"]:nth-child(7) > .rbc-button-link') to have no violations:

    <button type="button" class="rbc-button-link" role="cell">06</button>

    Received:

    "Certain ARIA roles must be contained by particular parents (aria-required-parent)"

    Fix any of the following:
      Required ARIA parent role not present: row

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.5/aria-required-parent?application=axeAPI
```

### Expected Behavior

I am receiving violations for basic accessibility testing

### Actual Behavior

I expect no violations to occur when running the test

### react-big-calendar version

1.5.2

### React version

18.2.0

### Platform/Target and Browser Versions

Chrome

### Validations

- [X] Read the [docs](http://jquense.github.io/react-big-calendar/examples).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a react-big-calendar issue and not an implementation issue

### Would you like to open a PR for this bug?

- [X] I'm willing to open a PR
